.github/actions/integration/entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh -l
 set -e
 
+# Setup github/worspace as safe
+git config --global --add safe.directory /github/workspace
```

Comment:
What's this?

Reply:
Recent got update regarding to security require this in our situation